modules/python/test/test_umat.py

```diff
@@ -12,22 +12,15 @@ def test_umat_construct(self):
         data = np.random.random([512, 512])
         # UMat constructors
         data_um = cv.UMat(data)  # from ndarray
-        data_sub_um = cv.UMat(data_um, [128, 256], [128, 256])  # from UMat
+        data_sub_um = cv.UMat(data_um, (128, 256), (128, 256))  # from UMat
         data_dst_um = cv.UMat(128, 128, cv.CV_64F)  # from size/type
         # test continuous and submatrix flags
         assert data_um.isContinuous() and not data_um.isSubmatrix()
         assert not data_sub_um.isContinuous() and data_sub_um.isSubmatrix()
         # test operation on submatrix
-        cv.multiply(data_sub_um, 2., dst=data_dst_um)
+        cv.multiply(data_sub_um, cv.UMat((128, 128), cv.CV_64F, 2.), dst=data_dst_um)
         assert np.allclose(2. * data[128:256, 128:256], data_dst_um.get())
 
-    def test_umat_handle(self):
```

Comment:
"Handle" was added to pass OpenCL buffer handles to other libraries like PyOpenCL.

It can't be removed.